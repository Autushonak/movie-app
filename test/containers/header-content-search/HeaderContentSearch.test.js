import 'jest-styled-components';
import React from 'react';
import { shallow } from 'enzyme';
import {
  HeaderContentSearchStyled,
  HeaderContentSearch,
} from '../../../src/containers/header-content-search/HeaderContentSearch';

describe('HeaderContentSearchStyled', () => {
  test('should be rendered correctly', () => {
    const component = shallow(<HeaderContentSearchStyled height={500} />);
    expect(component).toMatchSnapshot();
  });
});

describe('HeaderContentSearch', () => {
  test('should be rendered correctly', () => {
    const component = shallow(<HeaderContentSearch height={500} />);
    expect(component).toMatchSnapshot();
  });
});
