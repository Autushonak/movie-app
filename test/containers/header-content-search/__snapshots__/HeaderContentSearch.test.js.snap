// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderContentSearch should be rendered correctly 1`] = `
<styled.div
  height={500}
>
  <styled.h1>
    Find your movie
  </styled.h1>
  <SearchBar
    onClick={[Function]}
  />
  <ButtonGroup
    activeBtn="title"
    onClick={[Function]}
    title="Search By"
    values={
      Array [
        "title",
        "genre",
      ]
    }
  />
</styled.div>
`;

exports[`HeaderContentSearchStyled should be rendered correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  position: relative;
  width: 100%;
  height: 500px;
  z-index: 9999;
  padding: 0 150px;
  box-sizing: border-box;
}

<div
  className="c0"
  height={500}
/>
`;
